conda/recipes/rmm/meta.yaml

```diff
@@ -48,7 +48,6 @@ requirements:
     - python
     - scikit-build>=0.13.1
     - setuptools
-    - spdlog>=1.8.5,<2.0.0a0
```

Comment:
As far as I could tell the RMM cython doesn't directly use spdlog in any way other than needing to find it during build time because of the librmm dependency. It will get `spdlog` from the `run` dependency of the librmm recipe.